tests/test_io_stac.py

```diff
@@ -374,15 +374,12 @@ def test_feature_valid(rio):
             assert data.shape == (1, 119, 97)
 
 
-@patch("rio_tiler.io.cogeo.rasterio")
-def test_relative_assets(rio):
+def test_relative_assets():
```

Comment:
You don't need this `@patch` anymore?

Reply:
Ah I see it moved to `test_mosaic.py`

Reply:
well this is unrelated to the PR, I removed it because it wasn't needed, this test doesn't touch rasterio at all, just check that the links are 👌